coderd/database/dbtestutil/broker.go

```diff
@@ -0,0 +1,176 @@
+package dbtestutil
+
+import (
+	"database/sql"
+	_ "embed"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/lib/pq"
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/v2/cryptorand"
+)
+
+const CoderTestingDBName = "coder_testing"
```

Comment:
Probs should have the table as a const too

Reply:
It ends up being pretty annoying and potentially confusing to use such a constant. In statements like

```
	_, err = b.coderTestingDB.Exec(
		"INSERT INTO test_databases (name, process_uuid) VALUES ($1, $2)", dbName, b.uuid)
``` 

You can't use `$1` style substitution with Postgres for the table name, meaning that we'd have to put a `fmt.Sprint` in there. That creates 2 different kinds of token substitution in one statement, and might piss off linters about possible SQL injection.

I think the lack of clarity out weighs any gains we'd get from improved modifyability.